script/src/verify.rs

```diff
@@ -1124,7 +1131,12 @@ mod tests {
 
         assert_error_eq!(
             verifier.verify(100_000_000).unwrap_err(),
-            ScriptError::ValidationFailure(-1).output_type_script(0),
+            ScriptError::ValidationFailure(
+                "by-data-hash/342083c9f5e8f9bb866cb0ff1eafed2e29cfc7edd81e3e274c7b3b86a0fc519f"
```

Comment:
can we change the hardcoded hash value to `script.code_hash()`?

Reply:
Resolved in #2770